include/ibamr/INSVCStaggeredHierarchyIntegrator.h

```diff
@@ -463,6 +463,20 @@ class INSVCStaggeredHierarchyIntegrator : public INSHierarchyIntegrator
 
     /*!
      * Perform data initialization after the entire hierarchy has been constructed.
+     *
+     * \note If \f$ \nabla \cdot \mathbf{u} = Q \f$ is specified, then at the initial time
+     * the velocity projection step is omitted in this routine. This is because \f$ Q \f$ generally involves
+     * derivatives of transported variables, which are not known at the initial time. More concretely,
+     * imagine we are simulating a phase change or combustion process. the volume change effect
+     * \f$ \nabla \cdot \mathbf{u} \f$  that is captured by \f$ Q \f$ would be computed through
+     * gradient of temperature or species concentration. These gradients are not known at the
```

Comment:
through gradients of temperature or species concentration. 

or 

through the gradient of temperature or species concentration.

Reply:
I have done the changes and pushed it.